gi_core/CMakeLists.txt

```diff
@@ -19,22 +19,60 @@ project(gi_core)
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(common)
+include(CPM)
 
-find_c_and_cpp_files("${CMAKE_CURRENT_SOURCE_DIR}/include" gi_core_includes)
+# GI_CORE
+
+find_package(Boost COMPONENTS system filesystem unit_test_framework REQUIRED)
+find_package(OpenMP)
+
+CPMFindPackage(
+  NAME libsimdpp
+  GITHUB_REPOSITORY TheTryton/libsimdpp
```

Comment:
Why do we need a private fork ?

Reply:
https://github.com/TheTryton/libsimdpp/commit/7bdb659707a0d753918a7f6d75a581699ee2f34e. This one line doesn't compile for some reason (too long to explain) under the ubuntu gcc (mingw doesn't complain about it).